components/intake.py

```diff
@@ -0,0 +1,37 @@
+import ctre
+import wpilib
+import enum
+
+
+class Action(enum.Enum):
+    Intake = 1
+    Outtake = -1
+    Stop = 0
+
+
+class Intake:
+    """Bunny intake"""
+    intake_motor = ctre.CANTalon
+    intake_pdp_channel = int
+    pdp = wpilib.PowerDistributionPanel
+    # Max current in amps to draw before stopping the motor
+    max_current = 6
+    release_bunny = False
+    holding_bunny = False
+
+    def spit_bunny(self):
+        """Spit out a bunny if one is being held"""
+        self.release_bunny = True
+
+    def execute(self):
+        if self.holding_bunny:
+            if self.release_bunny:
+                self.intake_motor.set(-0.6)
```

Comment:
Let's make this full speed. The bag motor is pretty slow and weak, let's drop the bunny as fast as possible

Reply:
Just that one or the others as well?

Reply:
I’ll increase them all for now, we can tune it tomorrow.